components/authz-service/server/v2/project.go

```diff
@@ -45,12 +45,19 @@ func NewMemstoreProjectsServer(
 	ctx context.Context,
 	l logger.Logger,
 	e engine.ProjectRulesRetriever,
-	eventServiceClient automate_event.EventServiceClient,
-	configManager *config.Manager,
+	projectUpdateCerealManager *cereal.Manager,
 	pr PolicyRefresher,
 ) (api.ProjectsServer, error) {
 
-	projectUpdateManager := NewProjectUpdateManager(eventServiceClient, configManager)
+	//projectUpdateManager := NewProjectUpdateManager(eventServiceClient, configManager)
+	projectUpdateManager, err := RegisterCerealProjectUpdateManager(projectUpdateCerealManager,
+		"ProjectUpdate", "Runner", []string{
+			"ingest",
+			"compliance",
+		})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I am not sure I understand why there are two ways to create the `projectUpdateManager`. It looks like this one (`NewMemstoreProjectsServer`) is not being used.

Reply:
it might be a test thing, not sure. But if you have ideas on how you'd like to see this bit organized, i could make it happen. Right now, the domain services and authz need to agree on all of these values. Also, I need to not repeat this twice

Reply:
Oh, I see the `NewProjectUpdateManager` was already being called twice. 

Disregard my question.